src/state/store.tsx

```diff
@@ -1,29 +1,54 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { createStore, applyMiddleware } from 'redux';
 import thunkMiddleware from 'redux-thunk';
+import { persistStore, persistReducer } from 'redux-persist';
+import { createWhitelistFilter, createBlacklistFilter } from 'redux-persist-transform-filter';
+import storage from 'redux-persist/lib/storage';
+import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2';
 import rootReducer from './rootReducer';
 
+const persistConfig = {
+  key: 'root',
+  storage,
+  stateReconciler: autoMergeLevel2,
+  transforms: [
+    createWhitelistFilter(
+      'results',
+      ['tableColumns', 'viewMode'],  // save
+      ['tableColumns', 'viewMode'],  // load
+    ),
+    createBlacklistFilter(
```

Comment:
That's my understanding from reading the docs too, but there might be something wrong with my configuration because I can't get it work without using the black-list filter.

Reply:
As I couldn't find out why we need to have both black-list and white-list provided in the configuration object, this has to be addressed later on, possibly when I have to add the `basket` data to the React Persist.